crates/sui-indexer-alt-framework/src/cluster.rs

```diff
@@ -177,18 +177,18 @@ mod tests {
     use sui_synthetic_ingestion::synthetic_ingestion;
     use tempfile::tempdir;
 
-    use crate::db::{
-        temp::{get_available_port, TempDb},
-        Connection, Db,
-    };
+    use super::*;
```

Comment:
nit: `super::` was after `crate::` before because the import ordering goes from least to most specific: `std`, then other crates, then `crate`, then `super`, then `this`.